liquidity/parameters.go

```diff
@@ -373,7 +378,10 @@ func rpcToParameters(req *clientrpc.LiquidityParameters) (*Parameters,
 		SweepConfTarget: req.SweepConfTarget,
 		FailureBackOff: time.Duration(req.FailureBackoffSec) *
 			time.Second,
-		Autoloop:        req.Autoloop,
+		Autoloop: req.Autoloop,
+		AutoloopBudgetLastRefresh: time.Unix(
+			int64(req.AutoloopBudgetLastRefresh), 0,
```

Comment:
Yes, check this comment https://github.com/lightninglabs/loop/pull/556#issuecomment-1471014580

Reply:
Since we're exposing this as a parameter it should support both ways (read and write)

It could also be useful, e.g it's halfway through the month/year and user wants to set a monthly budget and wants it to line up with the start of the month/year, so they also define 1st of January (in unix) as the last budget refresh, with a period of 365days

In order to expose the last budget refresh but not allow overwriting it you would have to do some (kind of ugly) stuff like  always forcing the variable to stay the same, which would require some parsers to be `LiquidityManager` methods (in order to have access to the previous value).

Another thing to keep in mind is that the way this usually goes (AFAIU) from the RPC user's perspective is that they first do a `GetParameters` to retrieve the parameters object, then update the values they care about, then call `SetParameters` with the updated object. Unless you explicitly care for that field it should remain the same just like all the other "static" fields like rules/fees etc